### PR TITLE
enable append for file adapter on write

### LINKED ISSUE #286: add -append option to file adapter
at least for `jsonl` and `csv` this should be doable.

### PR DESCRIPTION
**Pull request by rlgomes**
fixes #286

simple change to make `-append` option available for the file adapter
which only works currently with csv and jsonl.

### CODE REVIEW

**Comment by rlgomes** [comment]
@davidvgalbraith @demmer check out my codez
@dmehra check out my english in the docs and the codez if you want too ;)

**Comment by dmehra** [comment]
+1 to the one line of English :)

File: test/adapters/file/write.spec.js [hunk 1]
```diff
@@ -111,4 +112,32 @@ describe('write file adapter tests', function () {
...
 
+    _.each(['csv', 'jsonl'], function(format) {
+        it('can append multiple points to a file with ' + format + ' format', function() {
+            return check_juttle({
+                program: 'emit -limit 1 ' +
+                         '| put value=1 ' +
+                         '| write file -file "' + tmp_file + '" -format "' + format + '"'
+            })
+            .then(function(result) {
+                expect(result.errors.length).to.equal(0);
```
Comment (davidvgalbraith) [inline test/adapters/file/write.spec.js]: `expect(result.errors).deep.equal([]);` is still preferred.

**Comment by davidvgalbraith** [comment]
+1 besides the `result.errors` validation.

**Comment by rlgomes** [comment]
fixed up the result.errors thing and will merge on green.

**Pull request merged**